src/containers/WaGroups/GroupDetails.tsx/GroupDetails.tsx

```diff
@@ -24,24 +24,12 @@ export const GroupDetails = () => {
   const [deleteVariables, setDeleteVariables] = useState<any>();
   const [groupDetails, setGroupDetails] = useState<any>();
 
-  // we should use wa_group api with filter by id here
-  const { loading: groupDataLoading } = useQuery(GROUP_SEARCH_QUERY, {
+  const { loading: groupDataLoading } = useQuery(GET_WA_GROUP, {
```

Comment:
```suggestion
  const { data: groupDetails, loading: groupDataLoading } = useQuery(GET_WA_GROUP, {
```

Use data object rather than creating another state variable